src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/Java/JavaMarshal.cs

```diff
@@ -0,0 +1,90 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Runtime.CompilerServices;
+using System.Runtime.Versioning;
+using System.Threading;
+
+namespace System.Runtime.InteropServices.Java
+{
+    [CLSCompliant(false)]
+    [SupportedOSPlatform("android")]
+    public static partial class JavaMarshal
+    {
+#if !NATIVEAOT
+        private static Thread? s_bridgeThread;
+#endif
+
+        public static unsafe void Initialize(
+            // Callback used to perform the marking of SCCs.
+            delegate* unmanaged<
+                nint,                               // Length of SCC collection
+                StronglyConnectedComponent*,        // SCC collection
+                nint,                               // Length of CCR collection
+                ComponentCrossReference*,           // CCR collection
+                void> markCrossReferences)
+        {
+#if NATIVEAOT
+            throw new NotImplementedException();
+#else
+            ArgumentNullException.ThrowIfNull(markCrossReferences);
+
+            if (!InitializeInternal((IntPtr)markCrossReferences))
+            {
+                throw new InvalidOperationException(SR.InvalidOperation_ReinitializeJavaMarshal);
+            }
+
+            s_bridgeThread = new Thread(BridgeMain)
+            {
+                IsBackground = true,
+                Name = ".NET GC Bridge Thread"
```

Comment:
```suggestion
                Name = ".NET GC Bridge"
```
IIRC, the native thread name length is limited to 15 characters on Linux: https://github.com/dotnet/runtime/blob/528dc0f2bb82669b4d555e2b5253211c204bd131/src/native/minipal/thread.h#L32

It looks better to fit the name into this limit, so that it is not cut-off in the middle of the word.